packages/kotti-ui/source/kotti-field-inline-edit/hooks.ts

```diff
@@ -0,0 +1,34 @@
+import { Ref, computed, watch } from '@vue/composition-api'
+
+import { KottiField } from '../kotti-field/types'
+
+import { KottiFieldInlineEdit } from './types'
+import { resizeTextArea } from './utils'
+
+export const useAdjustHeight = ({
+	field,
+	inputRef,
+	props,
+}: {
+	field: KottiField.Hook.Returns<KottiFieldInlineEdit.Value>
+	inputRef: Ref<HTMLTextAreaElement | HTMLInputElement | null>
+	props: KottiFieldInlineEdit.PropsInternal
+}) => {
+	if (!props.isMultiline) return
+	// Typecast needed because the type is logically implied from props.isMultiline being true
+	const _inputRef = inputRef as Ref<HTMLTextAreaElement | null>
```

Comment:
Swap this
```suggestion
	const inputRef = _inputRef as Ref<HTMLTextAreaElement | null>
```

This already caused line 25 to use the wrong variable